metadata-ingestion/scripts/install_odbc.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+UBUNTU_VERSION=$(grep VERSION_ID /etc/os-release | cut -d '"' -f 2)
+
+
+if ! [[ "20.04 22.04 24.04 24.10" == *"$UBUNTU_VERSION"* ]];
+then
+    echo "Ubuntu $UBUNTU_VERSION is not currently supported.";
+    exit;
+fi
+
+sudo su
+
+# Download the package to configure the Microsoft repo
+curl -sSL -O https://packages.microsoft.com/config/ubuntu/$UBUNTU_VERSION/packages-microsoft-prod.deb
```

Comment:
this code looks similar / duplicative to what we run in our ingestion dockerfiles? not a hard requirement, but would be nice to not have that duplication